dtlstransport.go

```diff
@@ -362,7 +349,8 @@ func (t *DTLSTransport) Start(remoteParameters DTLSParameters) error {
 	if err != nil {
 		t.onStateChange(DTLSTransportStateFailed)
 	}
-	return err
+
+	return t.startSRTP()
```

Comment:
I'm not that familiar with `dtlstransport.go`, but do we want to call `startSRTP` even when the err above is not nil?

Reply:
Oh nice catch! Oof bad mistake :(